src/v/kafka/server/handlers/fetch.cc

```diff
@@ -652,9 +653,7 @@ class simple_fetch_planner final : public fetch_planner::impl {
               };
 
               plan.fetches_per_shard[*shard].push_back(
-                make_ntp_fetch_config(ntp, config),
```

Comment:
Yeah, I did try compiling most commits individually but admittedly there has been a lot of rewriting since and I didn't always recompile.

So yeah I can squash those.